usagov/usagov-2021
# USAGOV-1802: asset refactor

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1802

## Description

Refactor SASS file to nest style selectors and organize them hierarchicaly
Move search script file from theme to module


## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


The code changes do not impact any style or functional changes.

### Change Requirements



### Validation Steps

- Homepage block, benefits nav block, and search pages should have same styles and functionality as before. I compared my local against https://federalist-e92b6227-c5cf-4fee-8448-9bef30fd37a7.sites.pages.cloud.gov/site/omerida/usagov-benefit-search/

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
